docs/resourse-cleanup.md

```diff
@@ -0,0 +1,64 @@
+# Stale Resource Cleanup
+
+If the PVC is created with storage class which is having the `reclaimPolicy``
+as `Retain` will not delete the PV object , backend omap metadata and backend image.
+Manual deletion of PV will result in stale omap keys, values , cephfs subvolume and rbd image.
+You need to cleanup metadata and image separately.
```

Comment:
Instead of 'you': 'It is required to cleanup ...'

Reply:
done.